Moab.Models/Helpers/ExerciseImportHelper.cs

```diff
@@ -20,17 +22,159 @@ public class ExerciseImportHelper
         public bool Import(string importCSV, ICollection<Exercise> exercises)
         {
             // TODO: Write the implementation!!!!!
+            //In my method UpdateExercise, I pass in a string array as the CSV file. This implies
+            //that at some point we create a method that splits the lines of CSV into arrays.
+            //However, we can't do this using the split method because of the commas in some hints.
+            List<string[]> ImportList = SplitCSVInput(importCSV);
+
+
             return false;
         }
 
         #endregion
 
+        #region test code for private methods  **REMOVE FROM PRODUCTION CODE**
+
+        public string[] TestLineSplit(string Line)
+        {
+            return SplitCSVLine(Line);
+        }
+
+        public List<string[]> TestInputProcessing(string input)
+        {
+            return SplitCSVInput(input);
+        }
+
+        public bool TestIsHeaderValid(string header)
+        {
+            return IsHeaderValid(header);
+        }
+
+        #endregion
+
         #region Private / Protected Methods
         protected bool IsHeaderValid(string header)
         {
+            string checkHeader = "ExerciseCode,Name,CDT_Class,CDT_AtHome,IsMovementDataCollected,UnitTarget,HintEasier,HintHarder,Hint1,Hint2,MDT_Class,MDT_AtHome,OldCode,Name_animationFile,Old_Name_animationFile";
+            if (header == checkHeader)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        protected Exercise FindExtantExerciseInCollection(ICollection<Exercise> exercises, string exerciseCode)
```

Comment:
You should add a comment that summarizes each of your methods. Visual Studio helps you do this if you first write the method signature (i.e. what the method returns, and what parameters are passed in to it), then above the line, type ///.   This generates a nicely-formed summary comment for methods.